generator/README.md

```diff
@@ -6,6 +6,12 @@ A tool for generating client libraries.
 
 Run the following command from the generator directory:
 
+```bash
+go run ./devtools/cmd/generate -language=rust
+```
+
+Alternatively, you can run the protoc command directly:
```

Comment:
Incidentally, with help from @codyoss I found a way to parse `.proto` files without having to call `protoc` at all.  Maybe we should pursue that for the generator. See #74.

Reply:
Nice!